Purchases/Attribution/RCAttributionFetcher.m

```diff
@@ -137,15 +137,32 @@ - (BOOL)isAuthorizedToPostSearchAds {
 
         BOOL needsTrackingAuthorization = [self.systemInfo isOperatingSystemAtLeastVersion:minimumOSVersionRequiringAuthorization];
 
-        Class<FakeATTrackingManager> _Nullable trackingManagerClass = [self.attributionFactory atTrackingManagerClass];
```

Comment:
not quite sure why, but `trackingManagerClass` won't have access to `methodSignatureForSelector` if the `<FakeATTrackingManager>` is there